dev/benchmarks/complex_layout/android/build.gradle

```diff
@@ -14,7 +14,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.2.0'
+        classpath 'com.android.tools.build:gradle:7.3.0'
```

Comment:
Out of curiosity, if there any reason why we couldn't start using a variable for version number like we do for the kotlin version?

Reply:
Agp version is more likely to cause you to have to make updates to your app than kotlin version or gradle version. Nothing mechanically stops us but I think leaving this version where it is easy for the Android Studio tooling to update is the best policy. That said it does probably make sense to have a version that can be shared for packages/packages.